google/ads/google_ads/client.py

```diff
@@ -588,11 +544,9 @@ def intercept_unary_unary(self, continuation, client_call_details, request):
             A grpc.Call/grpc.Future instance representing a service response.
```

Comment:
can it be grpc.Future?

Reply:
It's technically both, so I left it this way to be explicit.